aqua/Freecell/Portfile

```diff
@@ -14,19 +16,20 @@ description             classic solitaire game
 long_description        The imaginatively-titled “Freecell” is a Mac OS X \
                         version of the classic solitaire game.
 
-homepage                http://web.archive.org/web/20080622113332/http://randomoracle.com/freecell/
-master_sites            macports
-use_bzip2               yes
-
-checksums               md5 05c41aa87c3a6c390f83c12db7c4ff3d \
-                        sha1 2e609fe0548f44a316f77f15db39581aad7a587f \
-                        rmd160 0dbb3cbb23730654244091f4683c51a590a373ab
+checksums               md5 3171359a348ffa6ac46014a61235ef6b \
+                        sha1 f1218b7c1251fab5991201cd3fe2b48cb5e07673 \
+                        rmd160 df8ef0824ec660cac69a77a43c9dd24454a1a138 \
+                        size 1351501
 
 post-patch {
-    copy ${filespath}/Info-Freecell.plist ${worksrcpath}
-    move ${worksrcpath}/Freecell.pbproj ${worksrcpath}/Freecell.xcodeproj
-    copy -force ${filespath}/project.pbxproj ${worksrcpath}/Freecell.xcodeproj/project.pbxproj
+    platform darwin 8 {
+        reinplace "s|NSUInteger|unsigned|g" Card.m Game.h Game.m GameController.m GameView.m History.h History.m
+        reinplace "s|NSInteger|int|g" History.m HistoryController.m
+        reinplace "s|<NSTableViewDataSource>||g" History.h
+    }
 }
```

Comment:
Now I recently (yesterday) tried to slide through a one-character reinplace fix for a trivial typo in one file and a few hours later it was changed to a patchfile. So if a one-character trivial typo reinplace must be changed to a patchfile, you might expect wanting a patch for all this.

Of course, "policy" is that any significant changes are supposed to be patchfiles, not reinplaces...

Or who knows, perhaps this will be found to be fine in this case. Things do vary.